lisa/sut_orchestrator/azure/common.py

```diff
@@ -39,6 +41,10 @@
     Sku,
     StorageAccountCreateParameters,
 )
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.compute import ComputeManagementClient
+from azure.keyvault.secrets import SecretClient
+
```

Comment:
if it is a new package, please add it here refer https://github.com/microsoft/lisa/blob/main/pyproject.toml#L42

Reply:
Done! :)